cypress/support/pages/CommonPage.ts

```diff
@@ -0,0 +1,69 @@
+import Masthead from "./admin_console/Masthead";
+import SidebarPage from "./admin_console/SidebarPage";
+import TabPage from "./admin_console/components/TabPage";
+import FormPage from "./admin_console/components/FormPage";
+import ModalUtils from "../util/ModalUtils";
+import ActionToolbarPage from "./admin_console/components/ActionToolbarPage";
+import TableToolbarPage from "./admin_console/components/TableToolbarPage";
+import TablePage from "./admin_console/components/TablePage";
+import EmptyStatePage from "./admin_console/components/EmptyStatePage";
+
+export default class CommonPage {
+  private mastheadPage: Masthead;
+  private sidebarPage: SidebarPage;
+  private tabPage: TabPage;
+  private formPage: FormPage;
+  private modalPage: ModalUtils;
+  private actionToolbarPage: ActionToolbarPage;
+  private tablePage: TablePage;
+  private tableToolbarPage: TableToolbarPage;
+  private emptyStatePage: EmptyStatePage;
+
+  constructor() {
+    this.mastheadPage = new Masthead();
+    this.sidebarPage = new SidebarPage();
+    this.tabPage = new TabPage();
+    this.formPage = new FormPage();
+    this.modalPage = new ModalUtils();
+    this.actionToolbarPage = new ActionToolbarPage();
+    this.tablePage = new TablePage();
+    this.tableToolbarPage = new TableToolbarPage();
+    this.emptyStatePage = new EmptyStatePage();
+  }
+
+  masthead() {
+    return this.mastheadPage;
+  }
+
+  sidebar() {
+    return this.sidebarPage;
+  }
+
+  tab() {
+    return this.tabPage;
+  }
+
+  form() {
```

Comment:
```suggestion
  formUtils() {
```
Maybe it would be better to rename it, with `form` it seems like you can have the form as an object and get the required actions to fill data in a specific form, instead of just using the typical save, cancel buttons on one